recipe/build.sh

```diff
@@ -7,11 +7,16 @@ if [ "$(uname)" == "Darwin" ]; then
   export LDFLAGS="-rpath ${PREFIX}/lib ${LDFLAGS}"
   export LINKFLAGS="${LDFLAGS}"
   skiprpath="-DCMAKE_SKIP_RPATH=TRUE"
+  sed '340d' CMakeLists.txt > tmp
```

Comment:
what does this do?

Reply:
this is an hot fix for the fix we implemented with yarden to fix the segfault on OSX
it basically revert it


for a reason I ignore, the behavior of OX is different with and without conda.

That prove the fix we implemented is not good enough

Reply:
it remove line 340 and 342 of the CMakelist.txt:
`IF(NOT APPLE)` and `endif(NOT APPLE)`